### PR TITLE
INT 1726 Point Intake to Ferb For Home Page Screenings List

### PR DESCRIPTION
**Pull request by TylerOSI**
### Jira Story

- [Name of Story INT-1726](https://osi-cwds.atlassian.net/browse/INT-1726)

## Description
This change is to use Ferb to get the list of screenings on the home page from postgres -- vs elastic search through intake API & Ferb.

I also removed a virtus model and its factory which add unneeded complexity. We just need to pass JSON from the front end, through rails to calls out to APIs. The further we can simplify this process, the cleaner and easier to follow our code will be.

## Tests
- [ ] I have included unit tests 
- [ ] I have included feature tests 
- [ ] I have included other tests 
- [x] I have NOT included tests 

The current tests were changed to stub the new Ferb endpoint instead

## Types of changes

- [ ] New feature (non-breaking change which adds functionality)
- [x] Refactoring (No behavioral changes)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] I have ran all tests for the project.
- [x] I have ran lint/rubocop check for the changed/new files.
- [x] My code is in a stable state ready to be deployable, but not necessarily complete.
- [x] I promise on my honor as a CWDS developer to ensure I don't break the build, to check Lint/Rubocop, use best practices, to leave the code in better shape than I found it, and to have a good day.

### CODE REVIEW

**Comment by TylerOSI** [comment]
I may need to update some more feature specs, but I didn't want to run all specs locally and tie up my machine for a while.

File: app/controllers/api/v1/screenings_controller.rb [hunk 1]
```diff
@@ -86,10 +83,6 @@ def submit
 
       private
 
-      def screening_index_params
-        params.permit(screening_decisions: [])
```
[the remark this answers is not in the document]
Comment (TylerOSI) [reply app/controllers/api/v1/screenings_controller.rb]: Yeah that should be correct -- we aren't filtering screenings right now or providing any ability to search.

File: app/repositories/screening_repository.rb [hunk 2]
```diff
@@ -33,15 +33,13 @@ def self.update(security_token, screening)
     Screening.new(response.body)
   end
 
-  def self.search(security_token, search_terms)
-    response = IntakeAPI.make_api_call(
+  def self.search(security_token)
```
[the remark this answers is not in the document]
Comment (TylerOSI) [reply app/repositories/screening_repository.rb]: If it were a controller yeah that would make sense, but this is a repository so we can call it whatever we deem appropriate.

File: spec/features/home_page_spec.rb [hunk 3]
```diff
@@ -47,37 +47,37 @@
     end
 
     scenario 'includes a list of saved screenings' do
-      screening_one = FactoryBot.create(
```
Comment (NagaPradeepGajula-osi) [inline spec/features/home_page_spec.rb]: cool we are getting off using the factoryBot.

**Comment by TylerOSI** [comment]
retest this please

**Comment by TylerOSI** [comment]
retest this please

**Comment by TylerOSI** [comment]
retest this please

**Pull request merged**